plugins/polio/api.py

```diff
@@ -246,31 +246,50 @@ def get_year(current_date):
             return today.year
 
     @staticmethod
-    def get_calendar_data(self, year, params):
+    def get_calendar_data(self: Any, year: int, params: Any) -> Any:
+        """
+        Returns filtered rounds from database
+
+            parameters:
+                self (CampaignViewSet): a self CampaignViewSet
```

Comment:
The self parameter does need need explanation IMHO (given that we are in a non-static method)

Reply:
Fixed in : https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R285